examples/jobs/httpd.yaml

```diff
@@ -0,0 +1,15 @@
+name: httpd
+containers:
+- image: centos/httpd
+services:
+- name: httpd
+  type: NodePort
+  ports:
+  - port: 8080
+jobs:
+- containers:
```

Comment:
this is so confusing :(

So essentially we have `services` using `containers` as a root key, but jobs has `containers` within it?